### ISSUE TITLE
Unknown Subtitle Languare After Processing Bulk Items Using Queue Option

### ISSUE THREAD

**Issue by GoogleCodeExporter** [XeresRazor/subler#712]
```
First of all, let me start by saying that this is a great product, very well 
design and incredibly useful, it is by far the best option to add subtitles to 
mp4 files and one of the best for fetching metadata. 
Having said that, I would like to share a couple of issues/questions I’ve 
found while using the queue in order to process videos for adding subs and 
fetching metadata:
1.  It seems there is no option to choose the location of the subtitles to be 
used for the videos, right now the subs HAVE to be on the same folder of the 
videos. It would be nice to have the option to manually select the location of 
the subtitles folder.
2.  Another problem is that once the process is done, all added subtitles are in 
“unknown” language. An option to choose the bulk default language for the 
added subtitles and video would be highly appreciated, otherwise you have to 
manually get into the video and select the correct language for the subs and 
videos which defeats the whole purpose of using the automatic bulk processing.
3.  There is no option to select the metadata provider to be used for the 
fetching process, kind of what you have when using the “Search Metadata 
online…” 
I’m using Subler version 0.25 installed on OS X 10.9.1

```

Original issue reported on code.google.com by `israel.r...@gmail.com` on 15 Jan 2014 at 7:02
